.about.yml

```diff
@@ -111,6 +115,10 @@ services:
   category: security
   url: https://hakiri.io/github/18F/micropurchase/master
   badge: https://hakiri.io/github/18F/micropurchase/master.svg
+- name: Travis
+  category: Continuous Integration
+  url: https://travis-ci.org/18F/micropurchase
+  badge: https://api.travis-ci.org/18F/micropurchase.svg
```

Comment:
done

On Thu, Jul 28, 2016 at 3:51 PM, Jessie A. Young <notifications@github.com>
wrote:

> In .about.yml
> <https://github.com/18F/micropurchase/pull/971#discussion_r72717381>:
>
> > @@ -111,6 +115,10 @@ services:
> >    category: security
> >    url: https://hakiri.io/github/18F/micropurchase/master
> >    badge: https://hakiri.io/github/18F/micropurchase/master.svg
> > +- name: Travis
> > +  category: Continuous Integration
> > +  url: https://travis-ci.org/18F/micropurchase
> > +  badge: https://api.travis-ci.org/18F/micropurchase.svg
>
> maybe for this we can do master branch since that is what is on
> production? https://travis-ci.org/18F/micropurchase.svg?branch=master (I
> think this one you have will return badge for whatever the last travis run
> was...which means it will be red sometimes)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/micropurchase/pull/971/files/7708a8d134f1dc44821f72656055900725a20c38#r72717381>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGHNWi33BVtfkMcbefda14cmmD71gZ6ks5qaTJfgaJpZM4JXtbR>
> .
>



-- 
Michael Torres
Product Lead, Micropurchase Platform
18F
(415) 920-3205